style.css

```diff
@@ -4,18 +4,38 @@
  * @author dpogue
  */
 
-@import url(https://fonts.googleapis.com/css?family=Nova+Square&text=Plasma);
-@import url(https://fonts.googleapis.com/css?family=Cinzel);
-@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500,700);
+@import url('https://fonts.googleapis.com/css2?family=Nova+Square&display=swap&text=Plasma');
 
 @ms-viewport { width: device-width; }
 @viewport { width: device-width; }
 
+:root {
+    --page-background: #37372f;
+    --content-sidebar: url('pahtsBrdgTrim.png');
+    --content-background: #c6c3c6;
+    --content-foreground: #31454a;
+    --content-headings: #31454a;
+    --infobox-background: rgba(255, 255, 255, 0.25);
+}
+
+@media (prefers-color-scheme: dark) {
+    :root {
+        --page-background: #131311;
+        --content-sidebar: url('pahtsBrdgTrim-dark.png');
+        --content-background: #272527;
+        --content-foreground: #c2d2d6;
+        --content-headings: #e0e9eb;
+        --infobox-background: rgba(0, 0, 0, 0.25);
+    }
+}
+
+
 body {
-    background: #37372f; /*#5c5c5c;*/
+    background: #37372f;
+    background: var(--page-background);
     padding: 0;
     margin: 0 auto;
-    font-family: 'Roboto', sans-serif;
+    font-family: 'Roboto', system-ui, sans-serif;
```

Comment:
Roboto is no longer being imported.

Reply:
This will try to use Roboto if it's installed, falling back to the standard system sans-serif UI font (Segoe UI on Windows, San Francisco on macOS and iOS, Roboto on Android), and finally falling back to the default sans-serif browser font on other browsers (typically Arial).

This might result in minor font differences between browsers and operating systems, but it should be better for performance because it doesn't require loading additional fonts from another server.

Reply:
I'm aware of the fallback system, but I don't see why we can't retain the import for a freely-available font and ensure it's available if we're explicitly intending to use it.  What does system-ui and sans-serif default to on Android?

Reply:
`system-ui` and `sans-serif` should both be Roboto on Android. `sans-serif` will be Droid Sans on older versions of Android (pre 6.0 I think?)